svgpathtools/misctools.py

```diff
@@ -31,7 +31,7 @@ def rgb2hex(rgb):
     >>> rgb2hex((0,0,255))
     '#0000FF'
     """
-    return ('#%02x%02x%02x' % tuple(rgb)).upper()
+    return ('#%02x%02x%02x' % rgb).upper()
```

Comment:
The `tuple()` call is actually not redundant here -- it allows you to pass in e.g.`[255, 0, 0]`.

Reply:
I don't remember making that "tuple" change. Maybe it's something that got added later.

I'm changing it "back".

Regards stylistic changes such as "n+1" -> "n + 1". My linter actually complains of missing whitespace around binary operators such as +, -, etc. It says "PEP 8 (E226): missing whitespace around arithmetic operator".

So should we write "n+1" or "n + 1"?